lessons/codap-exploring/langs/en-us/pages/codap-exploration.adoc

```diff
@@ -1,50 +1,100 @@
 = CODAP Exploration
+++++
+<style>
+body.workbookpage .autonum { padding-top: 0.5rem; }
+body.workbookpage .fitb { padding-top: 0.7rem;}
+</style>
+++++
 
-Make sure you’re logged into the @starter-file{animals} in CODAP. The purpose of this exploration is to familiarize yourself with some of CODAP's features. Feel free to experiment and try things out!
+This page will help you familiarize yourself with some of CODAP's features. Check off each item once you have completed it. Feel free to experiment and try things out! Make sure you’re logged into the @starter-file{animals} in CODAP before beginning.
 
-@n See if you can make a graph. What did you select in order to do so?
+== Tables in CODAP
 
-@fitb{}{@ifsoln{I clicked on the "Graph" icon in the upper left-hand corner of my screen}}
+@n A table of data is shown here, with the title at the top of the table. What is the title? @fitb{}{@ifsoln{Animals-Dataset-1.5.1}}
 
-@fitb{}{}
+@n Move the table to a different location on the screen.
 
-@vspace{2ex}
+@n Minimize the table. (Hint: Try hovering your mouse over the table's title. What appears?)
 
-@n Move the table. Move the graph. How do you move things in CODAP?
+@n Now re-expand the minimized table.
 
-@fitb{}{@ifsoln{To move tables and graphs, I mouse over the top-most bar until my cursor turns into a hand. Then I can drag.}}
+@n Add a row to the table, then delete that same row.
 
-@fitb{}{}
+@n Move the `Age` column so that it is between `Fixed` and `Legs.`
 
-@vspace{2ex}
+@n Click on `Age` and, from the drop-down menu that appears, choose "Sort Ascending (A→Z, 0→9)." Now try "Sort Descending." 
 
-@n Minimize and maximize the table and the graph. How do you do that?
+@n How many animals have names that begin with S? @fitb{5em}{@ifsoln{6}}
 
-@fitb{}{@ifsoln{ To minimize, I select the `-` in the upper right-hand corner of the graph or table. To maximize, I reselect the `-`.}}
+@n Delete a column of the table.
```

Comment:
delete a column on the table (this is called an attribute - you use that word further down - suggest connecting the words right away)